src/lib/utils/hooks/SynapseAPI/entity/useGetQueryResultBundle.ts

```diff
@@ -23,7 +23,7 @@ import {
 import { entityQueryKeys } from './queryKeys'
 
 const sharedQueryDefaults = {
-  refetchOnWindowFocus: false,
+  staleTime: 1000 * 60 * 30, // 30 minutes
```

Comment:
For table queries, max out the stale time to match the cache time, i.e. don't auto-refetch data for 30 minutes

Reply:
Does this mean that after a Table is updated in SWC the user will not see the change for 30 minutes?

Reply:
They will see the change if they refresh the page (unless we move the cache to localStorage or sessionStorage), or if we invalidate the cache. The SWC EntityPresenter currently clears data for all entities from the cache when we fire the EntityUpdatedEvent.